HarpGuru/HarpFace/HarpCell/index.tsx

```diff
@@ -51,17 +51,22 @@ export const HarpCell = (props: HarpCellProps): React.ReactElement => {
   }
 
   const accessibleContent = 
-    <TouchableOpacity
+    <TouchableWithoutFeedback
       accessible={true}
       accessibilityRole='button'
-      style={styles.cell}
-      onPress={(): void => toggleActiveIdsIfHoleExists()} >
-      <Text>{displayValue}</Text>
-    </TouchableOpacity>
+      onPress={(): void => toggleActiveIdsIfHoleExists()}>
+      <View
+        style={styles.cell}>
+        <Text>
+          {displayValue}
+        </Text>
+      </View>
+    </TouchableWithoutFeedback>
 
   const inAccessibleContent = 
-    <View accessible={false} style={styles.cell}>
-    </View>
+    <TouchableWithoutFeedback disabled={true} accessible={false} style={styles.cell} >
```

Comment:
It would be good to have a test for the disabling of the components.

Reply:
This test already exists in a sense. The test for not having an accessibility role of 'button' fulfills this purpose.